frontend/public/components/app.jsx

```diff
@@ -159,6 +160,7 @@ class App extends React.PureComponent {
             </ConnectedNotificationDrawer>
           </Firehose>
         </Page>
+        <CloudShell />
```

Comment:
We should load this asynchronously because it's not needed right away.
```suggestion
        <AsyncComponent loader={() => import('@console/app/src/components/cloud-shell/CloudShell').then((m) => m.default)} />
```